tests/components/test-components.php

```diff
@@ -173,12 +173,13 @@ public function setUp() {
 	/**
 	 * Test output for core components.
 	 *
-	 * @param array  $expected Callback to set the expected component after hydration.
-	 * @param string $name     Name of the component being tested.
-	 * @param string $message  Optional. Message for failure.
+	 * @param array     $expected     Callback to set the expected component after hydration.
+	 * @param Component $component Name of the component being tested.
+	 * @param string    $message      Optional. Message for failure.
+	 *
+	 * @see \PHPUnit\Framework\Assert::assertEquals
 	 */
-	public function assertComponentEquals( array $expected, string $name, string $message = '' ) {
-		$component = new Component( $name );
+	public function assertComponentEquals( array $expected, Component $component, string $message = '' ) {
```

Comment:
It's easier to run assertions against an already instantiated component, rather than needing to pass in component arguments as well.